app/models/item.rb

```diff
@@ -1,4 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :merchant
   has_many :invoice_items
+
+  validates :name, presence: true, uniqueness: true
+  validates :description, presence: true
+  validates :unit_price, presence: true, numericality: { only_float: true}
+  validates :merchant_id, presence: true, numericality: {only_integer: true}
```

Comment:
Thank you for adding all of the validations, they will help everyone out as the project goes on!